modules/administration-guide/pages/assembly_configuring-oauth-1-for-bitbucket.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+:description: Configuring OAuth 1.0 for Bitbucket
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, bitbucket, oauth-1, oauth-1.0
+:navtitle: Configuring OAuth 1.0 for Bitbucket
+// :page-aliases:
+
+[id="configuring-oauth-1-for-bitbucket_{context}"]
+= Configuring OAuth 1.0 for Bitbucket
+
+To enable Che users to access a private Git repository that is hosted on a Bitbucket server:
```

Comment:
Do you use a product attribute for "Che?"

Reply:
:) We do of course. What an embarrassment.
```suggestion
To enable {prod-short} users to access a private Git repository that is hosted on a Bitbucket server:
```